src/unix/linux_like/linux/mod.rs

```diff
@@ -821,6 +821,41 @@ s! {
         pub val: ::c_int,
     }
 
+    // linux/ptp_clock.h
+    pub struct ptp_clock_time {
+        pub sec: ::__s64,
+        pub nsec: ::__u32,
+        pub reserved: ::__u32,
+    }
+
+
+
+    pub struct ptp_extts_request {
+        pub index: ::c_uint,
+        pub flags: ::c_uint,
+        pub rsv: [::c_uint; 2],
+    }
+
+    pub struct ptp_sys_offset_extended {
+        pub n_samples: ::c_uint,
+        pub rsv: [::c_uint; 3],
+        pub ts: [[ptp_clock_time; 3]; PTP_MAX_SAMPLES as usize],
+    }
```

Comment:
but our CI still uses the old version, right? So I'm not sure what to do with that right now

Reply:
It would probably be better to use the new version so we don't need to break things in the future. Just configure it to skip this struct in `build.rs`.